File: chatwoot/kotlin/src/main/kotlin/oseg/chatwoot_examples/DeleteAgentInInboxExample.kt
package oseg.chatwoot_examples

import com.chatwoot.client.infrastructure.*
import com.chatwoot.client.apis.*
import com.chatwoot.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class DeleteAgentInInboxExample
{
    fun deleteAgentInInbox()
    {
        ApiClient.apiKey["userApiKey"] = "USER_API_KEY"

        val deleteAgentInInboxRequest = DeleteAgentInInboxRequest(
            inboxId = null,
            userIds = listOf (),
        )

        try
        {
            InboxesApi().deleteAgentInInbox(
                accountId = null,
                _data = deleteAgentInInboxRequest,
            )
        } catch (e: ClientException) {
            println("4xx response calling InboxesApi#deleteAgentInInbox")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling InboxesApi#deleteAgentInInbox")
            e.printStackTrace()
        }
    }
}

File: chatwoot/typescript-node/src/DeleteAgentInInboxExample.ts
import * as fs from 'fs';
import api from "chatwoot_client"
import models from "chatwoot_client"

const apiCaller = new api.InboxesApi();
apiCaller.setApiKey(api.InboxesApiApiKeys.userApiKey, "USER_API_KEY");

const deleteAgentInInboxRequest: models.DeleteAgentInInboxRequest = {
  inboxId: "inbox_id_string",
  userIds: [
  ],
};

apiCaller.deleteAgentInInbox(
  0, // accountId
  deleteAgentInInboxRequest, // data
).catch(error => {
  console.log("Exception when calling InboxesApi#deleteAgentInInbox:");
  console.log(error.body);
});
